Lib/idlelib/help_about.py

```diff
@@ -88,8 +90,8 @@ def create_widgets(self):
         email = Label(frame_background, text='email:  idle-dev@python.org',
                       justify=LEFT, fg=self.fg, bg=self.bg)
         email.grid(row=6, column=0, columnspan=2, sticky=W, padx=10, pady=0)
-        docs = Label(frame_background, text='https://docs.python.org/' +
-                     python_version()[:3] + '/library/idle.html',
+        docs = Label(frame_background, text="https://docs.python.org/"
+                     f"{version[:version.rindex('.')]}/library/idle.html",
```

Comment:
It is not guaranteed that the Python version contains exactly 2 dots. This code will not work with versions containing 1 or 3 dots.

Reply:
See also `Lib/idlelib/editor.py`.

Reply:
> It is not guaranteed that the Python version contains exactly 2 dots.

Can you give an example where that's not the case?

Reply:
`3.2`? Also we can release a micro-bugfix in future, like `3.11.2.1`.

Reply:
OK, TIL that 2.7 and 3.2 were indeed missing the micro version in their initial releases. However:

1. We haven't done that since and even in those releases `sys.version_info` included `micro` equal to `0`.
2. The code in this PR uses `platform.python_version` which is specifically documented like this:

```
    """ Returns the Python version as string 'major.minor.patchlevel'

        Note that unlike the Python sys.version, the returned value
        will always include the patchlevel (it defaults to 0).
    """
```

3. We actually literally cannot release versions with a fourth component because no downstream tooling expects that. If we need to recall a version due to a big issue, we simply bump the third number.

Reply:
Good point. But all other code uses `sys.version_info[:2]` instead of parsing `platform.python_version()` if it needs a 2-component version. I think it is more reliable.